lib/DDG/Spice/Nutrition.pm

```diff
@@ -8,24 +8,21 @@ my $attribute_regex = qr/(?^:(?^:(?:c(?:a(?:l(?:ories(?: from fat)?|cium|s)|rb(?
 my $question_regex = qr/(?:how|what)?\s?(?:'s |is |are |many |much )?(?:the |there )?(?:total |amount of |number of )?/;
 
 triggers any => 'calories';
-
-# brand_id is hard coded to USDA for now. Eventually could support searches across brands (i.e. packaged goods or restaurants, but requires multiple
-# calls to their API so waiting for now):
-spice to => 'http://api.nutritionix.com/v1_1/search/$1?results=0%3A20&brand_id=513fcc648110a4cafb90ca5e&fields=*&appId={{ENV{DDG_SPICE_NUTRITIONIX_APPID}}}&appKey={{ENV{DDG_SPICE_NUTRITIONIX_APIKEY}}}';
 spice wrap_jsonp_callback => 1;
 
-handle query_lc => sub {
-    # check for queries like "how many calories are in a cucumber"
-    if (/^$question_regex$attribute_regex\s?(?:are |contained |is )?(?:there )?(?:in )?(?:a |an )?(.+?)(?:\?)?$/) {
-        return $1 if $1;
-    }
+# update for v2 (natural language): https://developer.nutritionix.com/docs/v2
 
-    # for queries like "cucumber calories" or "tofu how much protein"
-    if (/^(.+?)\s?$question_regex$attribute_regex(?:\?)?/) {
-        return $1 if $1;
-    }
+spice to => 'https://trackapi.nutritionix.com/v2/natural/nutrients';
+spice headers => {
+    "Content-Type" => "application/x-www-form-urlencoded",
+    "x-app-id" => "$ENV{DDG_SPICE_NUTRITIONIX_APPID}",
+    "x-app-key" => "$ENV{DDG_SPICE_NUTRITIONIX_APIKEY}"
+};
+spice post_body => 'query=$1';
 
-    return;
+handle query => sub {
+    # return the full query
```

Comment:
I'm not so sure iff that this is the problem. The example query here `calories in` is the measurement so `calories in banana` is like saying `how many calories in one calorie of a banana`. It's naive, but that's how the API handles that phrase 😕 

https://www.nutritionix.com/natural-demo?q=calories%20in%20banana&s=1

In my last commit, I have addressed this by ignoring the triggers and allow the API to infer the unit of measurement.